Test/test0/MaxKeepGoingSplits.bpl

```diff
@@ -1,10 +1,12 @@
+// First line checks for a deadlock
+// RUN: %boogie /vcsMaxKeepGoingSplits:2 /proverOpt:O:smt.qi.eager_threshold=100 "%s"
 // RUN: %boogie /vcsCores:3 /vcsMaxKeepGoingSplits:2 /trace /proverOpt:O:smt.qi.eager_threshold=100 "%s" > %t.log
 // RUN: %OutputCheck --file-to-check "%t.log" "%s"
 // CHECK-L: checking split 1/1, 0.00%
```

Comment:
Might it be better to make lines 5 and 6 consistent with 7-9 by dropping % and cost?